.spectral.yml

```diff
@@ -176,6 +176,63 @@ rules:
       functionOptions:
         notMatch: "https?://(localhost|127\\.0\\.0\\.1)(/|$)"
       invert: true
+  
```

Comment:
Förslag på mer kompakt form:

```json
"x-entur-permissions": {
  "all": [
    "organisations:read",
    {
      "any": [
        "items:read",
        "items-global:read"
      ]
    }
  ]
}
```

Eventuellt, men tror det är en dålig idé, en expressionsträng:

`"organisations:read AND (items:read OR items-global:read)"`

Reply:
Men `all`/`any` istedenfor `and`/`or` er jeg helt med på